src/locales/en/en.json

```diff
@@ -69,6 +69,12 @@
   "USER_EDIT_ROLE":"Edit User Role",
   "USER_ROLES":"Roles",
   "USER_ROLE_ADDED":"User Role Updated Successfully",
+
+  "USER_DASHBOARD_ROLE_NAME":"User Role",
```

Comment:
"USER_DASHBOARD_ROLE_NAME": "User Role",
"USER_DASHBOARD_USERS_ROLE_COUNT": "Count",
"USER_ROLE_DASHBOARD_LIST_WARNING": "Users Data Not Found",
"USER_DASHBOARD": "Dashboard",
"USER_DASHBOARD_ROLE": "Enter User Role",